force_nevergrad/mco/tests/test_ng_mco.py

```diff
@@ -72,14 +77,15 @@ def test_simple_run(self):
         model = self.factory.create_model()
         model.budget = 61
         model.parameters = self.parameters
-        model.kpis = [KPISpecification(), KPISpecification()]
+        model.kpis = [KPISpecification(scale_factor=1.0),
+                      KPISpecification(scale_factor=0.1)]
 
         workflow = Workflow()
         workflow.mco_model = model
-        kpis = [DataValue(value=1), DataValue(value=2)]
-        with self.assertTraitChanges(model, "event", count=61):
+        kpis = [DataValue(value=1), DataValue(value=5)]
+        with self.assertTraitChanges(model, "event", count=0):
```

Comment:
This is unexpected and may be the source of the unit test issues you mentioned. The `event` attribute here should be changed every time `model.notify_progress_event` is called in the `NevergradMCO.run` method (61 times for each KPI set calculated).

In which case it suggests the for loop in line 47 of `ng_mco.py` is not being executed properly